### PR TITLE
Rewrite ansible install documentation for Ubuntu/Debian-based devs

### PR DESCRIPTION
**Pull request by fowlslegs**

### CODE REVIEW

File: docs/development/getting_started.rst [hunk 1]
```diff
@@ -54,24 +54,34 @@ Either way, once you've installed Vagrant you should run:
...
 
     sudo apt-get install python-pip
-    sudo pip install -U ansible
 
-If you're using Ubuntu, you can install a sufficiently recent version of
-Ansible from backports (if you prefer): ``sudo apt-get install
-ansible/trusty-backports``
+Since the version of ansible required to provision SecureDrop VMs may flux out
+of sync with the version in your distro's repos, we recommend installing the
+latest stable ansible to a virtual environment using `virtualenvwrapper` or
```
Comment (garrettr) [inline docs/development/getting_started.rst]: "we recommend installing the latest stable ansible" No we don't, latest stable Ansible is 2.x which we currently don't support.

Comment (conorsch) [inline docs/development/getting_started.rst]: That's correct, we definitely _don't_ want to recommend 2.x—at least not yet. Let's simply leave the "stable" comment out and state the version, i.e. 

>  we recommend installing Ansible v1.8.4 in a virtualenv. 

Doesn't matter whether folks use `virtualenv` or `virtualenvwrapper`.

File: docs/development/getting_started.rst [hunk 2]
```diff
@@ -54,24 +54,34 @@ Either way, once you've installed Vagrant you should run:
...
+
+Using virtualenv (must be in SecureDrop base directory before running these
+commands):
+
+.. code:: sh
+
+    sudo apt-get install virtualenvwrapper
+    virtualenv -p python2.7 .
+    . bin/activate
+    pip install ansible==1.8.4
```
Comment (conorsch) [inline docs/development/getting_started.rst]: This example usage block doesn't actually use virtualenvwrapper, just plain old virtualenv. The wrapper version provides convenient aliases such as `mkvirtualenv` and `workon`.

The explicit version statement for the pip install is good, as is the removal of the sudo command.

File: docs/development/getting_started.rst (hunk 1 above)
Comment (fowlslegs) [inline docs/development/getting_started.rst]: Okay, 1.8.4 is the last tagged release in the ansible git repo. I'll just change it to say the latest stable from the first version.

Comment (conorsch) [inline docs/development/getting_started.rst]: > Okay, 1.8.4 is the last tagged release in the ansible git repo. 

No, v2.1.0.0-0.1.rc1 [is the latest](https://github.com/ansible/ansible/tags). Let's just say 1.8.4 and leave it at that.

File: docs/development/getting_started.rst (hunk 2 above)
Comment (fowlslegs) [inline docs/development/getting_started.rst]: Conor and I just talked about this in person, he did misread this section, but anyway, I do need to change the `apt-get` command in the second block to install virtualenv, not virtualenvwrapper.

File: docs/development/getting_started.rst (hunk 1 above)
Comment (fowlslegs) [inline docs/development/getting_started.rst]: Just updated then installed ansible on Tails 2.3, the latest stable, and they are shipping 1.7.2. Talking with Conor, he said he just installed SD on real hw using this version. It makes sense to test our playbooks against the version one would be using to do a real install, so I am going to rebase with that version and test it on my own workstation to make sure it works.

**Comment by fowlslegs** [comment]
I also noticed I meant to move that vagrant cachier warning, but only deleted it.

**Comment by fowlslegs** [comment]
Okay, can't test right now on my workstation, but will confirm works on sid w/ 1.7.2 (unless someone else wants to) later. Otherwise, it looks ready to merge.

**Comment by garrettr** [comment]
@fowlslegs It is not appropriate to merge changes to the developer documentation into a point release. Please rebase this onto develop. You will have to create a new pull request.

**Comment by garrettr** [comment]
The title of this pull request is bad. These changes hardly qualify as a rewrite.

Please also refrain from leaving the description of the pull request blank. At a minimum, there should be a description of the changes and the motivation behind them. Referencing existing issues, optionally using GitHub's auto-resolve syntax, is also nice.

**Comment by fowlslegs** [comment]
> The title of this pull request is bad.

Haha, that is true :-)

Closing, to re-pull against develop.

**Pull request closed without merging**